vendor/k8s.io/kubernetes/pkg/securitycontextconstraints/capabilities/mustrunas.go

```diff
@@ -101,9 +101,21 @@ func (s *defaultCapabilities) Validate(pod *api.Pod, container *api.Container) f
 		return allErrs
 	}
 
+	allowedAdd := makeCapSet(s.allowedCaps)
+	allowAllCaps := allowedAdd.Has(string(api.CapabilityAll))
+	if allowAllCaps {
+		if len(s.requiredDropCapabilities) > 0 {
```

Comment:
this validation doesn't belong here.  `s` here represents a strategy created from an `SCC` api object.  You have validation that ensures that this case should never happen in the api object itself.   

This should be for testing the pod/container requests against the ability to add everything.